charts/virtual-kubelet-for-aks/values.yaml

```diff
@@ -15,3 +15,12 @@ env:
   apiserverCert: 
   apiserverKey: 
   monitoredNamespace:
+
+# Install Default RBAC roles and bindings
+rbac:
+  install: true
+  serviceAccountName: virtual-kubelet
```

Comment:
serviceAccountName is not used in the deployment. I'm fine with using the fullname, but please remove this value to avoid confusion.